src/main/webapp/feed.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Message Feed</title>
+<link rel="stylesheet" href="/css/main.css">
+<link rel="stylesheet" href="/css/user-page.css">
+
+<script>
+    
+  // Fetch messages and add them to the page.
+  function fetchMessages(){
+    const url = '/feed';
```

Comment:
Can you format just the javascript part at https://prettier.io/playground/. Seems like the next line spacing between fetchMessages() and buildMessageDiv() is different.